### ISSUE TITLE
Raygun NullReferenceException

### ISSUE THREAD

**Issue by distantcam** [Particular/ServiceInsight#566]
NullReferenceException reported via Raygun for version 1.5.4+15.Branch.master.Sha.3a70dbbf7020d010778cb4b09542d09b33eadf18

```
ServiceInsight.Shell.ShellView.OnRestoreLayout(ISettingsProvider settingsProvider) in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\Shell\Shell.xaml.cs:85
 ServiceInsight.Shell.ShellViewModel.RestoreLayout() in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\Shell\ShellViewModel.cs:147
 Caliburn.Micro.ViewAware.Caliburn.Micro.IViewAware.AttachView(Object view, Object context):58
 Caliburn.Micro.ViewModelBinder.<.cctor>b__3(Object viewModel, DependencyObject view, Object context):170
 Caliburn.Micro.WindowManager.CreateWindow(Object rootModel, Boolean isDialog, Object context, IDictionary`2 settings):22
 Caliburn.Micro.WindowManager.ShowWindow(Object rootModel, Object context, IDictionary`2 settings):53
 ServiceInsight.Startup.AppBootstrapper.OnStartup(Object sender, StartupEventArgs e) in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\Startup\AppBootstrapper.cs:111
 System.Windows.Application.OnStartup(StartupEventArgs e):31
 ServiceInsight.App.OnStartup(StartupEventArgs e) in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\App.xaml.cs:54
 System.Windows.Application.<.ctor>b__1_0(Object unused):15
 System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs):119
 System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler):2
```

**Comment by distantcam** [comment]
Here's a stacktrace from 1.4.1+2.Branch.master.Sha.eab6d38808e52e5f757ae73f1480b280121ab0c1

```
Particular.ServiceInsight.Desktop.Shell.ShellView.OnRestoreLayout(ISettingsProvider settingsProvider) in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\Shell\Shell.xaml.cs:83
 Particular.ServiceInsight.Desktop.Shell.ShellViewModel.RestoreLayout() in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\Shell\ShellViewModel.cs:147
 Particular.ServiceInsight.Desktop.Framework.Rx.RxViewAware.Caliburn.Micro.IViewAware.AttachView(Object view, Object context) in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\Framework\Rx\RxViewAware.cs:62
 Caliburn.Micro.ViewModelBinder.<.cctor>b__3(Object viewModel, DependencyObject view, Object context) in c:\Users\Rob\Documents\CodePlex\caliburnmicro\src\Caliburn.Micro.Silverlight\ViewModelBinder.cs:193
 Caliburn.Micro.WindowManager.CreateWindow(Object rootModel, Boolean isDialog, Object context, IDictionary`2 settings) in c:\Users\Rob\Documents\CodePlex\caliburnmicro\src\Caliburn.Micro.WPF\WindowManager.cs:141
 Caliburn.Micro.WindowManager.ShowWindow(Object rootModel, Object context, IDictionary`2 settings) in c:\Users\Rob\Documents\CodePlex\caliburnmicro\src\Caliburn.Micro.WPF\WindowManager.cs:75
 System.Windows.Application.OnStartup(StartupEventArgs e):31
 Particular.ServiceInsight.Desktop.App.OnStartup(StartupEventArgs e) in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\App.xaml.cs:56
 System.Windows.Application.<.ctor>b__1_0(Object unused):15
 System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs):119
 System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler):2
```

**Comment by distantcam** [comment]
Fixed in 1.5.5
